tests/fixtures/identity.sql

```diff
@@ -822,3 +822,5 @@ SELECT NEXT VALUE FOR db.schema.sequence_name OVER (ORDER BY foo), col
 SELECT PERCENTILE_CONT(x, 0.5) OVER ()
 SELECT PERCENTILE_CONT(x, 0.5 RESPECT NULLS) OVER ()
 SELECT PERCENTILE_CONT(x, 0.5 IGNORE NULLS) OVER ()
+WITH my_cte AS (SELECT 'a' AS desc) SELECT desc AS description FROM my_cte
```

Comment:
does this break ordered?

select x from y order by x desc

Reply:
```python
>>> import sqlglot
>>> sqlglot.parse_one("select x from y order by x desc")
(SELECT expressions:
  (COLUMN this:
    (IDENTIFIER this: x, quoted: False)), from:
  (FROM this:
    (TABLE this:
      (IDENTIFIER this: y, quoted: False))), order:
  (ORDER expressions:
    (ORDERED this:
      (COLUMN this:
        (IDENTIFIER this: x, quoted: False)), desc: True, nulls_first: False)))
```

Reply:
Seems like `_parse_ordered` and `_parse_primary_key` are the only places where we reference `TokenType.DESC/ASC`. Let me take a closer look and I'll reply with an ACK if the execution flow looks ok.